### PR TITLE
lib: bsdlib: Add uptime functions

### PR DESCRIPTION
**Pull request by tommi-m**
Adds bsd_os_uptime_get and bsd_os_uptime_delta functions
to read system uptime.

Signed-off-by: Tommi Mammela <tommi.mammela@nordicsemi.no>

### CODE REVIEW

**Comment by rlubos** [comment]
Shouldn't this change come along with `bsdlib` and `bsd_os.h` header update?

**Comment by tommi-m** [comment]
> 
> 
> Shouldn't this change come along with `bsdlib` and `bsd_os.h` header update?

These are coming in next bsdlib release

**Comment by rlubos** [comment]
Sorry for the mess with reviewers, wrong window...

File: lib/bsdlib/bsd_os.c [hunk 1]
```diff
@@ -334,6 +334,16 @@ void bsd_os_errno_set(int err_code)
 	}
 }
 
+s64_t bsd_os_uptime_get(void)
+{
+    return k_uptime_get();
+}
+
+s64_t bsd_os_uptime_delta(s64_t *reftime)
```
Comment (evenl) [inline lib/bsdlib/bsd_os.c]: Is this for convenience or do we really need both the delta and the absolute?

Comment (tommi-m) [reply lib/bsdlib/bsd_os.c]: It's for convenience but I think it's goot to have both anyway.

**Comment by tommi-m** [comment]
BsdLib implementation changed. This is not needed any more.

**Pull request closed without merging**